src/js/components/products/notAllowed.tsx

```diff
@@ -35,17 +36,20 @@ const NotAllowed = ({
       </div>
     ) : null}
     <div className="slds-align_absolute-center">
-      <div className="slds-text-longform slds-text-body_regular">
-        {link} {i18n.t('or')}{' '}
-      </div>
-      <Login
-        id="product-not-allowed-login"
-        label={
-          isLoggedIn ? i18n.t('log in with a different org') : i18n.t('log in')
-        }
-        buttonClassName="slds-p-horizontal_xxx-small"
-        buttonVariant="base"
-      />
+      <Trans i18nKey="a_or_b">
+        <div className="slds-text-longform slds-text-body_regular">{link}</div>
+        {' or '}
```

Comment:
For spacing, the `{' or '}` should go inside the `div`. And <Trans> components cannot have non-string nodes as dynamic content:

```
Unsupported JSX expression. Only static values or {{interpolation}} blocks are supported. Got Identifier:
Node { type: 'Identifier', start: 1006, end: 1010, name: 'link' }
```